src/BaselineUtils.cc

```diff
@@ -488,7 +488,7 @@ _find_stray_flux(det::Footprint const& foot,
                 for (size_t i=0; i<tfoots.size(); ++i) {
                     contrib[i] = 0.0;
                 }
-                int i = nearest->get0(x, y);
+                int i = (*nearest)(x, y);
```

Comment:
You're being a bit inconsistent in whether you use `operator()(int, int)` or `operator[](Point2I)`. The difference seems to be `LOCAL` vs. `PARENT` coordinates, and I'm pretty sure `operator[]` (i.e., `PARENT`) is the right substitute.

Reply:
Thanks for catching that. I tried a few different iterations while debugging, and must have missed this one. Sadly, it looks like the tests couldn't tell the difference here.